test/tests/bootstrap/test_redfish10_os_bootstrap_parallel.py

```diff
@@ -0,0 +1,196 @@
+'''
+Copyright 2017, Dell EMC
```

Comment:
If you are making changes to this file, this is the Copyright from legal:
Copyright © 2017 Dell Inc. or its subsidiaries.  All Rights Reserved.